products/AOS-CX-Switching

```diff
@@ -0,0 +1,210 @@
+
+# What should be used to sort releases. Set to one of:
+# releaseCycle/eol/support/release/cycleShortHand/latest/latestShortHand
+# which must be present in the releases underneath
+sortReleasesBy: "releaseCycle"
+
+# Template to be used to generate a link for the release
+# __RELEASE_CYCLE__ will be replaced by the value of releaseCycle
+# __CYCLE_SHORT_HAND__ will be replaced by the optional changelogTemplate
+# __LATEST__ will be replaced by the value of latest
+# __LATEST_SHORT_HAND__ will be replaced by the optional latestShortHand
+# __CODENAME__ will be replaced by the optional codename
+
+# You can even use Liquid Templating inside the template, such as:
+# https://godotengine.org/article/maintenance-release-godot-__LATEST__
+# Do not use a localized URL (such as one containing en-us) if possible
+changelogTemplate: "https://link/of/the/__RELEASE_CYCLE__/and/__LATEST__/version"
+
+# Optional template that generates names for every release. Supports same templating as changelogTemplate.
+# Default value is `__RELEASE_CYCLE__``
+releaseLabel: "MoM Timeturner __RELEASE_CYCLE__ (__CODENAME__)"
+
+# The label that will be used alongside releases labelled with `lts: true`
+# Optional, only provide if the product has lts releases that are not called LTS, but something else.
+# Default Value is <abbr title='Long Term Support'>LTS</abbr>
+# Prefer using an HTML abbr tag, if possible.
+LTSLabel: "<abbr title='Extra Long Support'>ELS</abbr>"
+
+# Optional information about how release information can be fetched automatically
+# This is mainly used for the `latest` and `latestDate` fields of each release cycle
+# Please see https://github.com/endoflife-date/endoflife.date/wiki/Automation for more details
+# This is currently a WIP, and will not have any impact, but is highly recommended if this can be made available.
+auto:
+  # Any valid git clone URL will work
+  # Support for partialClone is necessary (GitHub does support this)
+  git: https://github.com/abc/def.git
+
+  # Valid OCI Image Registry URL
+  oci: https://index.docker.io/v2/_library/image
+
+  # Link to package on NPM
+  npm: https://www.npmjs.com/package/abc
+  # An optional regex that defines how the tags above should translate to releases
+  # Use named capturing groups
+  # Default value should work for most releases of the form a.b or a.b.c
+  # default also skips over any special releases (nightly,beta,pre,rc etc)
+  regex: ^v(?<major>0|[1-9]\d*)_(?<minor>0|[1-9]\d*)_(?<patch>\d{1,3})_?(?<tiny>\d+)?$
+  # A liquid template using the captured variables from the regex above that renders the final version
+  # You can use liquid templating here
+  template: '..'
+
+
+# A list of releases, supported or not
+# Newer releases go on top of the list, in order
+releases:
+  - releaseCycle: "10.09.xxxx"
+    releaseLabel: ""
+    eol: 2022-12-22
+    support: 2022-12-22
+    release: 2021-12-09
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
```

Comment:
This is same  just remove all cycleShortHand parts if they are empty. We are using cycleShortHand thing whenever
we need to correct sorting problems for products probably you wont need it any of those just remove all cycleShortHands